src/main/java/org/folio/circulation/domain/policy/lostitem/ChargeAmount.java

```diff
@@ -18,9 +17,12 @@ public ChargeAmount(BigDecimal amount, ChargeAmountType chargeType) {
   }
 
   public static ChargeAmount from(JsonObject chargeAmountType) {
-    return new ChargeAmount(
-      getBigDecimalProperty(chargeAmountType, "amount"),
-      forValue(getProperty(chargeAmountType, "chargeType")));
+    if (chargeAmountType == null) {
+      return null;
+    }
+
+    return new ChargeAmount(getBigDecimalProperty(chargeAmountType, "amount"),
+      forValue(chargeAmountType.getString("chargeType")));
```

Comment:
The no fee will be charged and loan won't be closed.

Reply:
Why would we want the loan to remain open if no fee is being charged?

Reply:
It likely will happen if there is some policy miss-configuration.

There is requirements to close the loan only if processing fee and SET_COST item fee is zero or processing fee and item fee is not defined at all (whole node is null).

When `chargeType` is missing but `amount` is not, we don't exactly now what chargeType is meant.

Reply:
I think this is a question for Holly. What should happen if the cost is partially defined, and the system cannot understand it.

Personally, I think if the system detects a misconfiguration, and does not know what to do, I think it should fail the process and not try to infer what it thinks the configuration should be.

(I think this ambiguity could've been avoided had the policy design been different, I don't really understand why charge type is needed at all)

Reply:
I'd personally postpone this question, because we only support set cost now, it is not clear how actual cost will be defined.

What the logic does now is creating fees/fines for the cases which exactly defined in requirements. 

I can create an issue for this question, so we can review it at some point. Does it make sense?